crates/redeem/src/tests.rs

```diff
@@ -517,6 +517,62 @@ fn test_cancel_redeem_succeeds() {
     })
 }
 
+#[test]
+fn test_mint_tokens_for_reimbursed_redeem() {
+    // Checked PRECONDITION: The vault MUST NOT be banned.
+    // Checked POSTCONDITION: `tryIncreaseToBeIssuedTokens` and `issueTokens` MUST be called,
```

Comment:
```suggestion
    // PRECONDITION: The vault MUST NOT be banned.
    // POSTCONDITION: `tryIncreaseToBeIssuedTokens` and `issueTokens` MUST be called,
```

Reply:
^ minor nit, but this is the style I adopted in the refund tests, cc: @theodorebugnet